features/telemetry/tally_metrics/README.md

```diff
@@ -0,0 +1,5 @@
+# Tally metrics
```

Comment:
I'm not convinced this belongs here.
I guess we could test a more general "metrics" concept where in TS you'd use sinks to check replay doesn't generate duplicate metrics.

Reply:
I am a bit stuck between the two. How do I demonstrate OpenTelemetry metrics with the Go SDK separate from Tally?

I think we accept that not all SDKs have all features when demonstrating specific ones. I am also planning on an OpenTracing feature, but I am not planning on calling it "tracing" just because it's not in TS.

However, in this case, if the library used for integration is too specific I can make a generic one. I just need to be able to demonstrate this one compared to OpenTelemetry metrics in separate features.

Reply:
I'm not convinced "Tally metrics" is a feature, "metrics" sounds more like a feature to me.
> I think we accept that not all SDKs have all features when demonstrating specific ones. I am also planning on an OpenTracing feature, but I am not planning on calling it "tracing" just because it's not in TS.
TS does support opentracing (with the otel shim).

Reply:
Yeah, I can see that. Is "OpenTelemetry metrics" a feature though? If I can figure out how to demonstrate _both_ metrics abstractions in a single Go feature, I don't mind making this generic.

Reply:
Changed to just "metrics"